arcaflow_plugin_rtla/rtla_plugin.py

```diff
@@ -157,7 +157,13 @@ def run_timerlat(
                 line.split()[0] in stats_names and not found_all
             ):
                 # Capture the columnar data
-                row_obj = dict(zip(col_headers, line.split()))
+                line_list = []
+                for element in line.split():
+                    try:
+                        line_list.append(int(element))
+                    except ValueError:
+                        line_list.append(element)
+                row_obj = dict(zip(col_headers, line_list))
```

Comment:
Your solution is more general, but I think it's simpler if we just handle the first column separately:
```suggestion
                items = line.split()
                row_obj = dict(zip(col_headers, map(int, items[1:])))
                row_obj[col_headers[0]] = items[0][:-1]
```
(Note that I've added code to remove the trailing colon from the first column's entry.)

Reply:
Nice. That's the clever approach I couldn't quite come up with late on Friday.